src/pages/en/core-concepts/astro-pages.md

```diff
@@ -145,6 +141,10 @@ export const get: APIRoute = ({ params, request }) => {
 
 ```
 
+## HTML Pages
+
+You can place a `.html` file in the `/src/pages/` directory. This will be treated as an [HTML Component](/en/core-concepts/astro-components/#html-components) and turned into a page.
```

Comment:
```suggestion
Files with the `.html` file extension can be placed in the `src/pages/` and used directly as pages on your site. Note that some key Astro features are not supported in [HTML Component](/en/core-concepts/astro-components/#html-components).
```

Kind of combining how we describe Astro pages with the link to seeing HTML components for the things Astro won't do with these files.